test/mocha.opts

```diff
@@ -0,0 +1,5 @@
+--compilers js:babel-register,jsx:babel-register
+--require airbnb-js-shims
+--require ./test/_helpers/ignoreSVGStrings.jsx
+--recursive
+test/**/*_spec.js test/**/*_spec.jsx test/_helpers/restoreSinonStubs.js
```

Comment:
however, the glob part of this configuration should remain in package.json, not in the mocha opts.